psi4/src/psi4/lib3index/dfhelper.h

```diff
@@ -105,6 +105,9 @@ class PSI_API DFHelper {
     void set_AO_core(bool core) { AO_core_ = core; }
     bool get_AO_core() { return AO_core_; }
 
+    /// Switch flag for releasing AO (for SAPT(DFT))
```

Comment:
The function `transform()` consists of 2 steps:
1. Transforms 3-index integrals with AO basis (pq|Q) to those with MO basis such as (ia|Q)
2. Contract it with a power of metric (since it will be useful in some density fitting algorithms). Most common choice of power would be -1/2 and this contraction would yield (ia|Q)(Q|Q')^(-1/2). 

In SAPT(DFT) hybrid dispersion, when in-core AO is turned on, the AO integral (pq|Q) is saved in the memory after step 1, but it is not needed in step 2, and releasing it allows much better blockings in step 2. (In fact what actually happened is that the memory limit was exceeded when trying to run large calculations before allowing the release of (pq|Q) after step 1)

However, it seems that when we release it, it will cause some other modules to fail, in particular Rob's FISAPT program for computing exchange-dispersion energy. This might be because in FISAPT (pq|Q) are still needed in step 2, but I have not figured out why this is the case. To avoid further complexity, I decided to release (pq|Q) only in SAPT(DFT) hybrid dispersion by using this switch flag.